src/shared/helpers/env.ts

```diff
@@ -1,8 +1,3 @@
-import { CustomWindow } from '../types/global';
-
-export function getEnv(name: keyof CustomWindow['_ENV_']): string | undefined {
```

Comment:
interface niet weg doen
mss 1 maal window casten naar CustomWindow en dan steeds gebruiken binnen de functie. Anders is er geen type checking meer